.travis.yml

```diff
@@ -39,6 +39,9 @@ matrix:
       env:

```

Comment:
is there a reason to run regression in both Cirrus CI and Travis CI?

I'm thinking that maybe running them in Cirrus CI should be enough. What do you think?

Reply:
That's my fault actually for not looking before acting. I went straight to add it to Travis CI and then I realized that you were using Cirrus CI. I will revert the changes for Travis CI then!